grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/AtomicBase.java

```diff
@@ -82,8 +83,13 @@ protected AtomicBase(AtomicBase a) {
     @Override
     public VarPattern getPattern(){ return atomPattern;}
 
+    protected Pattern combinedPattern(){ return atomPattern;}
```

Comment:
Maybe this should be `createCombinedPattern()` or something? We need to distinguish it from `getCombinedPattern()` below.